scenes/game/local_game.gd

```diff
@@ -1096,6 +1098,24 @@ class Player:
 		gauge = []
 		return events
 
+	func return_all_copies_of_top_discard_to_hand():
+		var events = []
+		var top_card = get_top_discard_card()
+		if not top_card:
+			return events
+
+		events += move_card_from_discard_to_hand(top_card.id)
```

Comment:
this line probably isn't necessary, right? the loop should take care of it

Reply:
uh, right.